### ISSUE TITLE
Name sorting: Omit component elements as well

### ISSUE THREAD

**Issue by bwiernik** [citation-style-language/documentation#136]
As noted here https://github.com/citation-style-language/documentation/pull/130

**Comment by denismaier** [comment]
I'm not sure. You mean if @form is "none" you will use an alternative name for sorting, but otherwise ignore it? Couldn't it be that you want to have publications written by author A as B grouped together, and also publications written by author A as C?

**Comment by bwiernik** [comment]
`form="none"` will generally be used to print the place without the publisher name. It makes sense to me that place, abbreviation, alternative should be dropped from when sorting, but when `form="none"`, that would mean an empty string remains. In that rare case, place, etc. should probably be kept for sorting.

**Comment by denismaier** [comment]
In most cases, yes. But what about the pseudonymn case?

**Comment by bwiernik** [comment]
Have a specific example? In most cases, I wouldn't expect a pseudonym/screen name/etc. to impact the sorting.

**Comment by denismaier** [comment]
No, no specific example. I've just thought it might be nice if you want works written by an author under a specific pseudonym to appear together, as a group so to say.

But, I've checked with the CMoS (14.80), and there  pseudonyms don't seem to impact the sorting. However, there are a couple of interesting regulations:

"A widely used pseudonym is generally treated as if it were the author's real name."

"The real name, if of interest to readers, may follow the pseudonym in brackets:"

"If the author's real name is better known than the pseudonym, the real name should be used. If needed, the pseudonym may be included in brackets."

14.81: "In some cases, a cross-reference from a real name to a pseudony, or vice versa, may be desired."

"If a bibliography includes two or more works published by the same author but under different pseudonyms,. all may be listed under the real name followed by the appropriate pseudonym in brackets, with cross-references under the pseudonym ... Alternatively, they may be listed under the pseudonyms, with a cross-reference at the real name to each pseudonym."

**Comment by bwiernik** [comment]
All of those cases I think would involve entering the real name (or whichever is the more common name) as the proper name and the other as the alternate. So I think we are good here.
